### PR TITLE
[dv/pwrmgr] Add feedback from testplan review

### PR DESCRIPTION
**Pull request by matutem**
Also fix minor glitches in the documentation.

Signed-off-by: Guillermo Maturana <maturana@google.com>

### CODE REVIEW

File: hw/ip/pwrmgr/doc/dv/index.md [hunk 1]
```diff
@@ -104,7 +103,11 @@ The `pwrmgr_reset_vseq` sequence checks the pwrmgr response to resets and reset
 
 The `pwrmgr_escalation_reset_vseq` sequence checks the response to an escalation reset.
 
-The `pwrmgr_reset_wakeup_vseq` sequence aligns reset and low power entry
+The `pwrmgr_reset_wakeup_vseq` sequence aligns reset and wakeup from low power.
+
+The `pwrmgr_lowpower_wakeup_race_vseq` sequence aligns a wakeup event coming
+in proximity to low power entry. Notice the wakeup is not expected to impact
```
Comment (sriyerg) [inline hw/ip/pwrmgr/doc/dv/index.md:109]: We have a one-sentence per line rule. Ditto below.

**Review by sriyerg — APPROVED** [review]
One full sentence per line, otherwise LGTM.

**Review by weicaiyang — APPROVED** [review]
LGTM

**Comment by matutem** [comment]
I made changes per the one sentence per line rule. build_docs works. Committing it now.

File: hw/ip/pwrmgr/doc/dv/index.md [hunk 2]
```diff
@@ -148,6 +150,12 @@ See also the test plan for specific ways these are driven to trigger different t
 - Input `main_pok` should turn on for the slow fsm to start power up sequence.
   Sequences will turn this off in response to `main_pd_n` going low, and turn it back on after a few random slow clock cycles from `main_pd_n` going high.
   Lack of a high transition causes a timeout, and would point to `main_pd_n` being set incorrectly.
+- Output transitions of `pwr_clamp_env` must always precede transitions of
+  `pwr_clamp` output.
+  Output transitions of `pwr_clamp` to active must always precede transitions
+  of `main_pd_n` output to active.
+  Output transitions of `pwr_clamp` to inactive must always follow transitions
+  of `main_pd_n` output to inactive.
```
Comment (sriyerg) [inline hw/ip/pwrmgr/doc/dv/index.md:158]: Missed these.

**Pull request merged**